lib/Basics/FileUtils.cpp

```diff
@@ -711,7 +711,7 @@ std::string slurpProgram(std::string const& program) {
   return std::string(buffer.data(), buffer.length());
 }
 
-int slurpProgramWithExitcode(std::string const& program, std::string &stdout) {
```

Comment:
agree that stdout is really bad name.  but I think this was copied.  do not remember writing this.